torch/utils/benchmark/op_fuzzers/unary.py

```diff
@@ -14,7 +25,9 @@
 
 
 class UnaryOpFuzzer(Fuzzer):
-    def __init__(self, seed, dtype=torch.float32, cuda=False):
+    def __init__(self, seed: Optional[int], dtype: _dtype | None = None, cuda: bool = False) -> None:
+        if dtype is None:
+            dtype = _DEFAULT_DTYPE
```

Comment:
```suggestion
            dtype = _DEFAULT_DTYPE
            assert dtype is not None
```
Instead of the weird TYPE_CHECKING stuff you are doing above.